tests/integration-tests/tests/iam/test_iam/test_iam_resource_prefix/pcluster.config.yaml

```diff
@@ -0,0 +1,49 @@
+Image:
+  Os: {{ os }}
+Iam:
+  ResourcePrefix: {{ iam_resource_prefix}}
+HeadNode:
+  InstanceType: {{ instance }}
+  Networking:
+    SubnetId: {{ public_subnet_id }}
+  Ssh:
+    KeyName: {{ key_name }}
+  Iam:
+    S3Access:
+      - BucketName: {{ bucket }}
+        KeyName: read_only/
+        EnableWriteAccess: false
+      - BucketName: {{ bucket }}
+        KeyName: read_and_write/
+        EnableWriteAccess: true
+Scheduling:
+  Scheduler: {{ scheduler }}
+  SlurmQueues:
+    - Name: queue-0
+      ComputeResources:
+        - Name: compute-resource-0
+          InstanceType: {{ instance }}
+          MinCount: {{ min_count }}
+      Networking:
+        SubnetIds:
+          - {{ private_subnet_id }}
+    - Name: queue-1
+      ComputeResources:
+        - Name: compute-resource-0
+          InstanceType: {{ instance }}
+          MinCount: {{ min_count }}
+      Networking:
+        SubnetIds:
+          - {{ private_subnet_id }}
+      Iam:
+        S3Access:
+          - BucketName: {{ bucket }}
+            KeyName: read_only/
+            EnableWriteAccess: false
+          - BucketName: {{ bucket }}
+            KeyName: read_and_write/
+            EnableWriteAccess: true
+DevSettings:
+  Timeouts:
+    HeadNodeBootstrapTimeout: 1234  # timeout in seconds
```

Comment:
Why do we need to set the headnode bootstrap timeout for these tests?

Reply:
Will remove them.